static/tabs/home.html

```diff
@@ -0,0 +1,223 @@
+                            </div>
+                        </div>
+                    </div>
+                    <div class="card-header fs-6">Print Progress</div>
+                    <div class="card-body">
+                        <div class="row mb-3 g-3">
+                            <div class="col-12">
+                                <div class="input-group">
+                                    <div class="input-group-text print-info-label" id="printName">Name</div>
+                                    <span
+                                        id="print-name"
+                                        class="form-control"
+                                        title="Print Name"
+                                        aria-describedby="printName"
+                                    >
+                                    </span>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="row mb-3 g-3">
+                            <div class="col-6">
+                                <div class="input-group">
+                                    <div class="input-group-text print-info-label" id="timeElapsed">
+                                        <i class="icon icon-time-passed"></i>
+                                    </div>
+                                    <span
+                                        id="time-elapsed"
+                                        class="form-control text-center"
+                                        title="Time Elapsed"
+                                        aria-describedby="timeElapsed"
+                                    >
+                                        00:00:00
+                                    </span>
+                                </div>
+                            </div>
+                            <div class="col-6">
+                                <div class="input-group">
+                                    <div class="input-group-text print-info-label" id="timeRemain">
+                                        <i class="icon icon-time-remaining"></i>
+                                    </div>
+                                    <span
+                                        id="time-remain"
+                                        class="form-control text-center"
+                                        title="Time Remaining"
+                                        aria-describedby="timeRemain"
+                                    >
+                                        00:00:00
+                                    </span>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="row mb-3 g-3">
+                            <div class="col-12">
+                                <div class="progress position-relative" id="printProgress">
+                                    <div
```

Comment:
With the change of the color for the progress bar, we should probably change the font color so that there is greater contrast. 

![image](https://github.com/Ankermgmt/ankermake-m5-protocol/assets/3013565/be4edef5-b280-46e9-93ae-9ae673bda360)

Reply:
good call on that, that almost reminds me of a colorblind challenge.

Reply:
Fixed :white_check_mark: